docs/aggregation-api.md

```diff
@@ -0,0 +1,14 @@
v.432h-6.228c.054 1.386.738 2.232 2.286 2.232 1.062 0 1.8-.432 1.944-1.206h1.98c-.378 2.052-2.232 2.7-3.96 2.7zm-2.214-5.076h4.212c-.162-1.224-1.062-1.746-2.106-1.746-1.044 0-1.944.522-2.106 1.746zm9.414 4.896v-7.956h1.674v1.08c.378-.522 1.116-1.26 2.466-1.26 1.512 0 2.844.648 2.844 3.222V245h-1.926v-4.536c0-1.296-.27-1.998-1.44-1.998-.864 0-1.35.342-1.692.72V245h-1.926zm13.95 3.456c-1.764 0-3.582-.486-3.636-2.988h1.854c.054.936.486 1.53 1.854 1.53 1.656 0 1.854-.882 1.854-1.944v-1.134c-.378.396-1.098.954-2.322.954-1.854 0-3.384-1.242-3.384-3.978 0-2.754 1.584-4.032 3.384-4.032 1.296 0 2.196.648 2.574 1.206v-1.026h1.674v8.172c0 2.142-1.224 3.24-3.852 3.24zm.18-5.166c.756 0 1.332-.288 1.746-.684v-3.456a2.467 2.467 0 0 0-1.746-.702c-1.386 0-1.98.99-1.98 2.43 0 1.44.594 2.412 1.98 2.412zm11.07 1.89c-2.304 0-4.194-1.134-4.194-4.122 0-3.186 2.124-4.194 4.086-4.194 2.034 0 4.086 1.008 4.086 4.158v.432h-6.228c.054 1.386.738 2.232 2.286 2.232 1.062 0 1.8-.432 1.944-1.206h1.98c-.378 2.052-2.232 2.7-3.96 2.7zm-2.214-5.076h4.212c-.162-1.224-1.062-1.746-2.106-1.746-1.044 0-1.944.522-2.106 1.746zM43.108 408v-10.296h3.384c2.484 0 4.392 1.026 4.392 3.402 0 2.394-1.89 3.402-4.392 3.402h-1.386V408h-1.998zm1.998-8.604v3.438h1.404c1.602 0 2.304-.63 2.304-1.728 0-1.098-.702-1.71-2.304-1.71h-1.404zm11.034 8.784c-1.548 0-2.808-.684-2.808-2.268 0-1.836 1.638-2.322 3.276-2.322.828 0 1.71.144 2.178.288v-.648c0-1.026-.18-1.836-1.656-1.836-1.116 0-1.764.45-1.764 1.386h-1.818c0-2.178 1.782-2.916 3.708-2.916 2.07 0 3.456.882 3.456 3.114v3.294c0 .306.108.414.432.414.216 0 .54-.054.756-.108v1.278a4.128 4.128 0 0 1-1.53.324c-1.008 0-1.296-.522-1.296-1.116v-.144c-.414.522-1.296 1.26-2.934 1.26zm.45-1.44c1.008 0 1.8-.342 2.196-.666v-1.008c-.342-.09-1.08-.162-1.764-.162-1.062 0-1.764.198-1.764.954 0 .612.45.882 1.332.882zm7.992 1.26v-1.512h1.962v-4.914h-1.962v-1.53h3.636v1.8c.342-.882 1.134-1.98 2.7-1.98.594 0 1.134.144 1.494.342v1.944c-.396-.234-1.008-.378-1.53-.378-1.26 0-2.016.63-2.412 1.152v3.564h3.348V408h-7.236zm15.66.18c-2.052 0-3.132-.792-3.132-3.564v-3.042h-2.538v-1.53h2.538v-3.69h1.926v3.69h3.906v1.53h-3.906v2.97c0 1.494.288 2.016 1.62 2.016.738 0 1.422-.162 1.98-.396v1.548c-.576.252-1.332.468-2.394.468zm5.4 3.186v-1.584c1.728 0 2.574-.54 3.15-1.836l-3.528-7.902h2.16l1.53 3.762.882 2.448.9-2.43 1.548-3.78h2.052l-3.33 7.56c-1.224 2.754-2.394 3.762-5.364 3.762zM95.92 408l3.618-10.296h2.124L105.28 408h-2.142l-.756-2.268h-3.618L98.026 408H95.92zm3.636-4.914l-.342 1.062h2.664l-.342-1.062-.99-3.33-.99 3.33zm8.46 4.914l-1.386-7.956h1.818l.828 6.426 1.134-4.842h2.016l1.17 4.842.81-6.426h1.764L114.784 408h-2.25l-1.134-4.572-1.134 4.572h-2.25zm12.924.18c-1.548 0-2.808-.684-2.808-2.268 0-1.836 1.638-2.322 3.276-2.322.828 0 1.71.144 2.178.288v-.648c0-1.026-.18-1.836-1.656-1.836-1.116 0-1.764.45-1.764 1.386h-1.818c0-2.178 1.782-2.916 3.708-2.916 2.07 0 3.456.882 3.456 3.114v3.294c0 .306.108.414.432.414.216 0 .54-.054.756-.108v1.278a4.128 4.128 0 0 1-1.53.324c-1.008 0-1.296-.522-1.296
```

Comment:
By saying "the orchestrator", it sounds like there's one function or something that's holding on to the party and dealer states, but they could be on different computers entirely, with the orchestration code handling sending messages over the network.

Reply:
👍